### ISSUE TITLE
Java runtime cannot be located

### ISSUE THREAD

**Issue by WritingSolutions2020** [Microsoft/vscode#45095]
For what it's worth, as of 05 March, 2018 the problem illustrated in https://github.com/Microsoft/vscode-java-debug/issues/112 still exists. The screenshot in that issue illustrates the problem.

For Linux Mint 32-bit. Visual Studio code 1.20.1.

1. Installed `/java/jdk1.8.0_161` in `/usr` directory. In `/bin` used `java -version` to verify that the Java executable could be run without error.

2. Added path to JDK in both user and workspace settings: 

**User Settings:**
```
{
   "java.home": "/usr/java/jdk1.8.0_161",
}
```
**Workspace Settings:**
```
{
  "java.home": "/usr/java/jdk1.8.0_161",
  "python.pydev.java.home": "/usr/java/jdk1.8.0_161",
  "python.linting.pylintPath": "/usr/local/bin/pylint",
  "python.pythonPath": "/usr/bin/python3.5",
}
```
3. Although I'm creating Python 3.5 code and not Java code, installed the following extensions in response to guidance offered on this and other forums:

    ![visualstudiocode12installedextensions](https://user-images.githubusercontent.com/3333327/37005322-8e209326-2088-11e8-8b85-c2cbe37c48dc.png)

See also https://stackoverflow.com/questions/46751168/visual-studio-code-cant-find-jdk8

4. Although I have resolved other issues (such as the inability of Visual Studio Code to find the system's instance of _pylint_) the JDK problem persists. And as others with this problem have noted both in the Github issue cited above and elsewhere, the solutions offered haven't worked.

**Comment by WritingSolutions2020** [comment]
This may be an issue with the PyDev extension rather than VCS itself. Closing...

**Comment by andxu** [comment]
Lets track it through https://github.com/Microsoft/vscode-java-debug/issues/112

**Comment by EryetChen** [comment]
i have the same issue after the recent update(before update everything works fine)

**Comment by WritingSolutions2020** [comment]
A bit more information. The problem I'm having is that it appears the NONE of the extensions that require a JRE / SDK work. This despite the fact that I've (1) configured and tested JAVA_HOME and PATH, and (2) carerfully configured VCS's JSON configurations.

With regard to JRE / SDK, I have other programs -- Eclipse, jEdit, and others -- that require a JRE. They are all working properly after I installed and pointed them to _jdk1.8.0_161_. I've also used `java -version` in various places to ensure that that SDK is now this computer's default.

Also, VCS does not report the error until I (a) have PyDev installed, and (b) open a Python 3 source file. So the central problem appears to me to be that PyDev can't use the information I've carefully provided to enable it to find the SDK it needs. AND after removing the PyDev extension the "JRE not found" error no longer occurs.
